app/src/main/java/com/example/keybindhelper/MainActivity.kt

```diff
@@ -55,28 +55,29 @@ class MainActivity : AppCompatActivity() {
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
 
-        DatabaseManager.init(binding.root.context);
+        DatabaseManager.init(binding.root.context)
         CurrentProjectManager.loadFirstProject()
 
 
     }
 
     fun showMenuItems(items:Set<Int>){
+        val allActionMenuIds= setOf(R.id.action_add,R.id.action_delete_all_groups,R.id.action_show_hide_keybinds)
         allActionMenuIds.forEach{
-            this.Menu?.findItem(it)!!.isVisible = items.contains(it)
+            this.menu?.findItem(it)!!.isVisible = items.contains(it)
```

Comment:
Its better practice to begin function names with lowercase
source: https://kotlinlang.org/docs/coding-conventions.html#function-names